docs/ref/models/database-functions.txt

```diff
@@ -163,6 +163,32 @@ and ``comment.modified``.
     The PostgreSQL behavior can be emulated using ``Coalesce`` if you know
     a sensible minimum value to provide as a default.
 
+``JSONArray``
+--------------
+
```

Comment:
```suggestion

.. versionadded:: 5.2

```
This also needs a 5.2 release note

Reply:
Thanks. None of the other functions in that module had a versionadded, so I just didn't think about it.

Reply:
We need a release note for Django 5.2 :+1:

Reply:
I've added a note. Just copy/pasted the release note for JSONObject and tweaked it.